### PR TITLE
succinct bullets for ONCOHIST

### PR DESCRIPTION
**Pull request by gregquinn2001**
For JIRA 1965. Also removed onco hist build methods plus other app layer methods from data access layer FluxConditionPresent. Keep people from searching and updating the wrong method and reduce the amount of code we have!

_Pull requests into Flux require the following. Submitter should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:. Reviewers should complete every item on the bulleted list when reviewing._

## Submitter:

**Running the Application**

- [X] Manually tested in Chrome (primary browser for testing)
- [ ] Manually tested in IE/Safari (verify app loads and basic feature testing)
- [ ] Manually tested in Chrome in iPad mode (at minimum, viewing patient record)

**Task**

- [X] Branch implements task as expected
- [X] Key application features are updated as needed for task and tested (see: [Key Application Features](https://github.com/FluxNotes/flux/wiki/Key-Application-Features))
- [X] Task specific definition of done (as described in the Jira task) is met, if applicable.

**Documentation**

- [X] This pull request describes why these changes were made
- [X] Recognizes any potential shortcomings/bugs in the description 
- [X] Pre release script is updated on the [Pre Release Testing Script wiki page](https://github.com/FluxNotes/flux/wiki/Pre-Release-Testing-Script) (add/remove single features to test, do not run through full script)
- [X] Update the list of key features with new features on the [Key Application Features wiki page](https://github.com/FluxNotes/flux/wiki/Key-Application-Features)
- [X] Document any new APIs/extension points
- [X] Additional technical components and libraries are documented and added to [wiki](https://github.com/FluxNotes/flux/wiki/About-Flux-Notes) as needed

**Code Quality**

- [X] Code style guide is followed (see: [Code Style Guide](https://github.com/FluxNotes/flux/wiki/JavaScript-and-HTML-Code-Style-Guide))
- [X] Code is commented

**Tests**

- [X] Existing tests passed
- [X] Added backend tests for new functionality added

## Reviewers:

- Check that code is maintainable and reusable. Code reuses existing code and infrastructure where appropriate.
- Check the PR and code accomplishes the task's purpose.
- Check that new tests appropriately test the new code, including edge cases. Other existing tests pass.
- Verify Pull Request checklist is correctly completed by submitter.
- Try to break the code

### CODE REVIEW

File: src/model/base/FluxConditionPresentAssertion.js [hunk 1]
```diff
@@ -480,18 +480,18 @@ class FluxConditionPresentAssertion extends FluxEntry {
...
                     if (event.asOfDate && event.status) {
-                        hpiText += `\r\n- As of ${event.asOfDate}, disease is ${event.status}`;
+                        hpiText += `\r\n- ${event.status} as of ${event.asOfDate}`;
                         if (event.evidence && event.evidence.length > 0) {
-                            hpiText += ` based on ${event.evidence.join(', ')}.`;
-                        } else {
-                            hpiText += '.';
+                            hpiText += ` based on ${event.evidence.join(', ')}`;
+                        // } else {
+                        //     hpiText += '.';
```
Comment (jafeltra) [inline src/model/base/FluxConditionPresentAssertion.js]: Do we need these commented out lines anymore?

**Comment by Dtphelan1** [comment]
This looks pretty good. The only question I have is if there's anything I should look at to make sure that removing all those methods from the data-source v-1 FluxConditionPresentAssertion didn't break any functionality? I'm doing a manual scan now, but was wondering if there was a testing strategy already tried that I should use.

**Comment by jafeltra** [comment]
This looks good to me as well. I have the same question as Dylan, and I also noticed a few lines that were commented out that I think can just be removed. I can push a commit to remove them if you'd like.

**Comment by gregquinn2001** [comment]
I removed commented out code. Talk to Dan about testing if you need to. You need to use a data source that uses a hard-coded mCODE 0.1 file to test that data source. Look in AppManager for an endpoint that uses that data source

**Comment by jafeltra** [comment]
I also noticed one content question about the oncohist text.

[JA1] The imatinib bullet should probably have some sort of punctuation (maybe a period, maybe a comma) separating out the started medication information and the stopped medication information. It seems like we aren't using periods/sentences for most bullets, so maybe the comma is the better approach. However, the bullet above has different sentences (the surgery bullet). We should probably make sure these align.

**Comment by Dtphelan1** [comment]
I looked over the endpoints that use the v01DataSource, and they _feel_ fine, but it feels like this was a big additional change unrelated to the small PR we were trying to get in under the wire. I'm the last person to complain about refactoring code as we go along, The time crunch just makes for a tough circumstance.

**Comment by danlee1025** [comment]
It looks like the V01 FluxConditionPresentAssertion is only imported in the EntryMapper and doesn't use any of the removed functions to do the mapping so I don't think it should break anything.

**Comment by Dtphelan1** [comment]
Other than JA1 I'm good to approve!

**Comment by gregquinn2001** [comment]
JA1 addressed

**Comment by gregquinn2001** [comment]
Sorry Dylan. It's a good point. I knew it was pretty safe and that if it did break something it wouldn't be pilot1 or ccp. You are correct though that I probably should not have done it since we were trying to rush it in. In the end, we are taking the time it takes as we should though so we are ok. I love eliminating code!

**Comment by gregquinn2001** [comment]
Thanks @danlee1025 !

**Pull request merged**